ca/ca.go

```diff
@@ -432,13 +432,19 @@ func (ca *CertificateAuthorityImpl) IssuePrecertificate(ctx context.Context, iss
 		return nil, err
 	}
 
+	var regID int64
```

Comment:
tiny nit: it seems lie you could avoid the pre-declaration of `regID` and instead change L439 to be just: `regID := *issueReq.RegistrationID`